### PR TITLE
✨ Implements amp-sona custom component

### PR DESCRIPTION
**Pull request by daniel-samson**
Implements an extension for the amp-sona component to the AMP project. This relates to the Intention to implement #29013.

### CODE REVIEW

**Comment by daniel-samson** [comment]
@googlebot I fixed it.

File: extensions/amp-sona/0.1/amp-sona.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (kristoferbaxter) [inline extensions/amp-sona/0.1/amp-sona.js:2]: Can you update this value to `2020`? The value should match the year the new file is committed.

File: extensions/amp-sona/0.1/test/test-amp-sona.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (kristoferbaxter) [inline extensions/amp-sona/0.1/test/test-amp-sona.js:2]: Can you update this value to 2020? The value should match the year the new file is committed.

File: extensions/amp-sona/0.1/test/validator-amp-sona.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (kristoferbaxter) [inline extensions/amp-sona/0.1/test/validator-amp-sona.html:2]: Can you update this value to 2020? The value should match the year the new file is committed.

File: extensions/amp-sona/0.1/test/validator-amp-sona.out [hunk 4]
```diff
@@ -0,0 +1,43 @@
+PASS
+|  <!--
+|    Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (kristoferbaxter) [inline extensions/amp-sona/0.1/test/validator-amp-sona.out:3]: Can you update this value to 2020? The value should match the year the new file is committed.

File: extensions/amp-sona/amp-sona.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+$category@: ads
+formats:
+  - websites
+teaser:
+  text: Displays sona widget.
+---
+
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (kristoferbaxter) [inline extensions/amp-sona/amp-sona.md:10]: Can you update this value to 2020? The value should match the year the new file is committed.

File: extensions/amp-sona/validator-amp-sona.protoascii [hunk 6]
```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (kristoferbaxter) [inline extensions/amp-sona/validator-amp-sona.protoascii:2]: Can you update this value to 2020? The value should match the year the new file is committed.

File: extensions/amp-sona/0.1/amp-sona.js (hunk 1 above)
Comment (daniel-samson) [reply extensions/amp-sona/0.1/amp-sona.js]: 👍

File: extensions/amp-sona/0.1/test/validator-amp-sona.html (hunk 3 above)
Comment (daniel-samson) [reply extensions/amp-sona/0.1/test/validator-amp-sona.html]: 👍

File: extensions/amp-sona/amp-sona.md (hunk 5 above)
Comment (daniel-samson) [reply extensions/amp-sona/amp-sona.md]: 👍

File: extensions/amp-sona/validator-amp-sona.protoascii (hunk 6 above)
Comment (daniel-samson) [reply extensions/amp-sona/validator-amp-sona.protoascii]: 👍

File: extensions/amp-sona/0.1/test/validator-amp-sona.out (hunk 4 above)
Comment (daniel-samson) [reply extensions/amp-sona/0.1/test/validator-amp-sona.out]: 👍

File: extensions/amp-sona/0.1/test/test-amp-sona.js (hunk 2 above)
Comment (daniel-samson) [reply extensions/amp-sona/0.1/test/test-amp-sona.js]: 👍

**Comment by daniel-samson** [comment]
Opting to modify amp-ad instead of creating an extension

**Pull request closed without merging**